test/source/patterns.ts

```diff
@@ -81,11 +81,12 @@ for (const srcFilePath of getAllFilesInDir('./extension', /\.ts$/)) {
 const expectedPermissions: chrome.runtime.ManifestPermissions[] = ['alarms', 'scripting', 'storage', 'tabs', 'unlimitedStorage'];
 const expectedConsumerHostPermissions = ['https://*.google.com/*', 'https://www.googleapis.com/*', 'https://flowcrypt.com/*'];
 const expectedEnterpriseHostPermissions = ['https://*.google.com/*', 'https://*.googleapis.com/*', 'https://flowcrypt.com/*'];
-for (const buildType of ['chrome-consumer', 'chrome-enterprise']) {
+for (const buildType of ['chrome-consumer', 'chrome-enterprise', 'thunderbird-consumer', 'firefox-consumer']) {
   const manifest = JSON.parse(readFileSync(`./build/${buildType}/manifest.json`).toString()) as chrome.runtime.Manifest;
+  const webBrowserBuilds = buildType.includes('chrome') || buildType.includes('firefox');
   const expectedHostPermissions = buildType.includes('consumer') ? expectedConsumerHostPermissions : expectedEnterpriseHostPermissions;
   for (const expectedHostPermission of expectedHostPermissions) {
-    if (!manifest.host_permissions.includes(expectedHostPermission)) {
+    if (webBrowserBuilds && !manifest.host_permissions.includes(expectedHostPermission)) {
       console.error(`Missing host permission '${expectedHostPermission}' in ${buildType}/manifest.json`);
       errsFound++;
     }
```

Comment:
let's check for permissions in thunderbird builds too, it'll be just `manifest.permissions` array instead of `manifest.host_permissions`:
```ts
const buildHostPermissions = webBrowserBuilds ? manifest.host_permissions : manifest.permissions;
if (!buildHostPermissions.includes(expectedHostPermission)) {
...
}
```

Reply:
Thanks @sosnovsky - I have added a test for this one for the thunderbird build. Would you mind checking it again? Thank you.